src/disks.py

```diff
@@ -0,0 +1,49 @@
+import dataclasses
+from typing import Optional
+
+import dbus
+from moulinette.utils.log import getActionLogger
+
+logger = getActionLogger("yunohost.storage")
+
+
+UDISK_DRIVE_PATH = "/org/freedesktop/UDisks2/drives/"
+UDISK_DRIVE_IFC = "org.freedesktop.UDisks2.Drive"
+
+
+@dataclasses.dataclass
+class DiskInfos:
+    name: str
+    model: str
+    serial: str
+    size: int
+    type: str
+    rpm: Optional[int] = None
+
+
+def infos():
+    result = {}
+
+    bus = dbus.SystemBus()
```

Comment:
Should it be file-scoped? I don't know how it is efficient to create a bus at each call.

Reply:
but maybe dbus is ok... (i am not a dbus expert)